CHANGELOG

```diff
@@ -1,6 +1,8 @@
 v3.4.8 (2019-09-XX)
 -------------------
 
+* Drop collection action to timeout more quickly to stay on fast lane
```

Comment:
Decreased timeout for drop collection action to stay on fast lane.

Reply:
Sure, would change, if you insist, but the current version is perfect and arguably better English